.circleci/config.yml

```diff
@@ -261,6 +266,60 @@ jobs:
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat; cd habitat-sim
               python examples/example.py --scene data/scene_datasets/habitat-test-scenes/van-gogh-room.glb --silent --test_fps_regression $FPS_THRESHOLD
+      - add_ssh_keys:
+          fingerprints:
+            - "18:88:e0:37:b1:b3:7a:23:aa:1e:f7:43:a8:5b:8e:05"
+      - run:
+          name: Update documentation
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat;
+              cd habitat-sim
+              # Rebuild with all options enabled
+              ./build.sh --with-cuda --with-bullet
+              cd docs
+              conda install -y -c conda-forge doxygen
+              conda install -y  jinja2 pygments docutils
+              conda install -y -c conda-forge pelican
+              mkdir -p ../build/docs
+              git submodule update --init
+              doxygen
+              ./build-public.sh
+              cd ../..
+
+              # Update website
+              git clone git@github.com:facebookmicrosites/habitat-website.git
+              cd habitat-website
+              git submodule update --init
+
+              for dir in habitat-sim habitat-cpp
+              do
+                  rm -rf published/docs/${dir}
+                  cp -r ../habitat-sim/build/docs-public/${dir} published/docs/.
+              done
+              git checkout published/docs/habitat-sim/index.html
```

Comment:
`index.html` will never be updated?

Reply:
It will be update only manually as we have hardcoded links to tutorials. cc @mosra if I understood incorrectly.

Reply:
Huh, no, it shouldn't behave like that, at all. I wasn't aware of https://github.com/facebookmicrosites/habitat-website/commit/f16a3d1c932a8b9ed1f6f48ef99a9f269ce0a909 -- it's behaving correctly on my side, generating links as intended (testing with current master). I bet it's due to some unfortunate minor difference in docutils on the CI?

In any case, the following patch should make it work correctly, as it explicitly uses the `:ref:` role instead of relying on the overriden default role:

```diff
diff --git a/docs/pages/index.rst b/docs/pages/index.rst
index b2fe3e8..c1907aa 100644
--- a/docs/pages/index.rst
+++ b/docs/pages/index.rst
@@ -14,9 +14,9 @@ over 10,000 FPS multi-process on a single GPU!
 
 .. TODO: this is waiting on m.css to propagate page titles to links
 
--   `New Actions <std:doc:new-actions>`
--   `Notebooks <std:doc:notebooks>`
--   `Creating a stereo agent <std:doc:stereo-agent>`
+-   :ref:`New Actions <std:doc:new-actions>`
+-   :ref:`Notebooks <std:doc:notebooks>`
+-   :ref:`Creating a stereo agent <std:doc:stereo-agent>`
 
 .. note-warning::
 
```

@mathfac can you confirm the above makes it work correctly, without the `git checkout` needed?